meta/main.yml

```
@@ -6,12 +6,10 @@
 ---
 galaxy_info:
   role_name: "rsd"
-  namespace: "hifis"
   author: "HIFIS Software team"
```

```suggestion
  author: "hifis"
```

It might be a try to see if this fixes the pipeline error. @Normo